sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts

```diff
@@ -264,12 +264,10 @@ export abstract class ECertFileHandler extends ESDCFileHandler {
       postalCode: addressInfo.postalCode,
       email: student.user.email,
       gender: student.gender,
-      // TODO: getting the information directly from the student profile since there is no
-      // direction from the business at this moment from where it should actually come.
-      ppdFlag: student.studentPDVerified,
       maritalStatus: application.relationshipStatus,
       studentNumber: application.studentNumber,
       awards,
+      ppdFlag: application.data.applicationPDPPDStatus,
```

Comment:
applicationPDPPDStatus?

Reply:
@dheepak-aot I didn't understand this comment. Any typo?

Reply:
I meant the property name 
![image](https://github.com/bcgov/SIMS/assets/54600590/054757af-85fc-48ed-b1d9-e96f3ba08afc)